web/examples/range-series.html

```diff
@@ -84,7 +84,9 @@
                 });
             } else if (!rangesEqual(chart.filter(), overviewChart.focusChart().filter())) {
                 dc.events.trigger(function () {
-                    overviewChart.focusChart().focus(chart.filter());
+                    // The second parameter is quite important. It asks the focus operation not to propagate events
+                    // In absence of that it will cause infinite mutual recursion
+                    overviewChart.focusChart().focus(chart.filter(), true);
```

Comment:
Have you considered defaulting this the other way? I'm not clear which circumstances lead to infinite recursion, or which way is more backward-compatible

Reply:
I will actually love to name the 2nd parameter in `_chart.focus` to `raiseEvents`  instead of `noRaiseEvents`. To use default value `false`, I had to settle for that name.